tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>this intro notebook is too long, what about moving the rest to another one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Reply:
<p>Sounds good, the rest of the notebook is not changed from the original- just a bucket of utility documentation.</p>

Reply:
<p>I moved these utilities to a separate <code>intro_utilities.ipynb</code>.</p>